SparkleSpin/PlayerViewModel.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PlayerViewModel.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 2/27/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+import Foundation
+
+class PlayerViewModel {
+    
+    private let playerModel: PlayerModel
+    var nameString: String? = nil
+    
+    init(playerModel: PlayerModel) {
+        self.playerModel = playerModel
+    }
+    
+    func updateProperties(playerModel: PlayerModel, nameString: String) {
```

Comment:
What's the reasoning for passing the Model here again? Looks like you are already passing it in the initializer.

Reply:
After revisiting this, I can see why this is redundant. I will only pass the `nameString` here.